docs-v2/snippets/generated/zoominfo/PreBuiltUseCases.mdx

```diff
@@ -0,0 +1,5 @@
+## Pre-built use-cases
+
+_No pre-built use-cases yet (time to contribute: &lt;48h)_
+
+<Tip>Not seeing the use-case you need? [Build your own](https://nango.dev/slack) independently.</Tip>
```

Comment:
You could remove 239 files from the repo by reusing a component when it's empty or creating a symlink

Reply:
No sure I get your message. But I guess since it's currently a "one time" generator you can discard